src/slurm_plugin/suspend.py

```diff
@@ -122,14 +89,14 @@ def _suspend(arg_nodes, suspend_config):
     slurm_nodes = get_nodes_info(arg_nodes)
     log.debug("Slurm_nodes = %s", slurm_nodes)
 
-    instance_ids_to_nodename = _get_instance_ids_to_nodename(
-        slurm_nodes, suspend_config.region, suspend_config.cluster_name, suspend_config.boto3_config
+    terminate_associated_instances(
```

Comment:
let's make sure we log somewhere the instance ids of the instances we are trying to terminate and ideally the associated slurm node

Reply:
Added log entry in `terminate_associated_instances`